app/components/ui/atoms/icons/Search.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default props => (
+  <svg viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
+    <g fill="none" fillRule="evenodd" stroke="none" strokeWidth="1">
+      <g
+        fill="#888888"
+        id="molecules/search-input"
+        transform="translate(-255.000000, -16.000000)"
+      >
+        <path d="M267.5,27 L266.7,27 L266.4,26.7 C267.4,25.6 268,24.1 268,22.5 C268,18.9 265.1,16 261.5,16 C257.9,16 255,18.9 255,22.5 C255,26.1 257.9,29 261.5,29 C263.1,29 264.6,28.4 265.7,27.4 L266,27.7 L266,28.5 L271,33.5 L272.5,32 L267.5,27 L267.5,27 Z M261.5,27 C259,27 257,25 257,22.5 C257,20 259,18 261.5,18 C264,18 266,20 266,22.5 C266,25 264,27 261.5,27 L261.5,27 Z" />
+      </g>
+    </g>
+  </svg>
+)
```

Comment:
Please put all SVGs through https://svgr.now.sh. For example the above becomes: 
```
<svg viewBox="0 0 18 18" {...props}>
    <path
      d="M12.5 11h-.8l-.3-.3c1-1.1 1.6-2.6 1.6-4.2C13 2.9 10.1 0 6.5 0S0 2.9 0 6.5 2.9 13 6.5 13c1.6 0 3.1-.6 4.2-1.6l.3.3v.8l5 5 1.5-1.5-5-5zm-6 0C4 11 2 9 2 6.5S4 2 6.5 2 11 4 11 6.5 9 11 6.5 11z"
      fill="#888"
    />
</svg>
```

Reply:
When was this agreed upon? Is there an ADR about it?

Reply:
Haha! I mean if you really, really don't like that tool you don't have to use it but it just seems to make sense to commit clean and tidy SVGs.

Reply:
I used that tool and the SVG I get doesn't look like the original icon, at least on the page which is weird. If I save both as svg files and open them with an image editor they both look the same to me, but when I load them on the search bar they look different. I attached images with both cases.
![converted-search-svg](https://user-images.githubusercontent.com/1497469/43717151-84dd3ff2-997e-11e8-9e5c-242eda68d92a.png)
![search-svg](https://user-images.githubusercontent.com/1497469/43717153-87fad6ae-997e-11e8-919e-fd58ce8ef04a.png)

Reply:
Ok, so it seems to happen because the converter forgot about `stroke='none'` attribute. Open new issue on their repo? :)

Reply:
I think the attributes are a little messed up on the original svg though. It has both `stroke="none"` and `strokeWidth="1"`.

Reply:
It's because you're wrapping the icon in the `Icon` component which applies a stroke to it. By default there is no stroke so the SVG is correct.

Reply:
It seems that the Icon component only sets the stroke color. I'm not sure if in the absence of `stroke="none"` this will force a stroke or not. If that's the case then it makes sense what happened, since the original svg has `stroke="none"` which will make it disappear.

What I was saying about the original SVG is that there is no need to set the stroke width if it's nonexistent.